frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -67,6 +67,8 @@
 
         vm.subErrors = {};
 
+        vm.privateLeaderboardUnique = 0;
```

Comment:
`vm.privateLeaderboardUnique` -> `vm.isChallengeLeaderboardPrivate = false`

Reply:
@MarchingVoxels Can you please change the name of the variables as stated above.